common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -126,6 +132,9 @@ public boolean setValue(Object value) {
     }
 
     public boolean isUserEdited() {
+        if (this.metadata.internal()) {
+            return true;
+        }
         return userEdited;
     }
```

Comment:
whether or not this gets changed according to my other comment, this method should probably get renamed to something more universal, like `shouldSave`